cypress/integration/seededFlows/articleFlows/postModerationTools.spec.js

```diff
@@ -41,10 +41,22 @@ describe('Moderation Tools for Posts', () => {
     });
   });
 
-  it('should not alter tags from a post if a reason is not specified', () => {
+  it("should not show moderation badge for a person's own posts", () => {
     cy.fixture('users/adminUser.json').as('user');
     cy.get('@user').then((user) => {
       cy.loginAndVisit(user, '/admin_mcadmin/tag-test-article').then(() => {
+        cy.findByRole('button', { name: 'Moderation' }).should('not.exist');
```

Comment:
It's good practice to wait for _something_ to exist on a page before asserting that something doesn't exist (because almost nothing exists on the page when you first visit it and it's still loading, making it possible to get false passes)

Since this button also only ever appears async, I think we are _still_ pretty likely to get false positives on this test even if we wait for the page heading to appear - the mod button is presented in the UI async so when you first visit a post page _everyone_ is missing the moderation button for a some small amount of time.

There isn't an obvious moderation-related element to 'wait for' to make sure the page is finished loading, but given we're looking at the user's own post, I think we could say "wait until we at least know this is the current user" by checking the "Follow" button has changed to "Edit profile". I haven't tested this out locally, but I think it should work:

```suggestion
        cy.findByRole('heading', { level: 1, name: 'Tag test article' });
        // Make sure the page has completed initializing
        cy.findByRole('button', { name: 'Edit profile' });
        cy.findByRole('button', { name: 'Moderation' }).should('not.exist');
```